gen/src/main.rs

```diff
@@ -121,18 +131,18 @@ impl crate::IconShape for Shape {
 "#;
 
 const PATH_TEMPLATE: &str = r#"
-Shape::{NAME} => rsx! {
-    path {
-        {ATTRS}
-    },
-},"#;
+            Shape::{NAME} => rsx! {
+                path {
+                    {ATTRS}
+                },
+            },"#;
 
 fn write_icons_file(icons: &[Icon], to: &PathBuf) {
     let names = icons
         .iter()
         .map(|i| i.name.as_str())
         .collect::<Vec<_>>()
-        .join(",\n");
+        .join(",\n    ");
```

Comment:
All of the generated code will be cleaned up with `rustfmt` anyway so there's no need to worry about whitespace in that code.